spec.txt

```diff
@@ -5354,9 +5354,18 @@ well.  ([reStructuredText](http://docutils.sourceforge.net/rst.html)
 takes a different approach, requiring blank lines before lists
 even inside other list items.)
 
-In order to solve of unwanted lists in paragraphs with
-hard-wrapped numerals, we allow only lists starting with `1` to
-interrupt paragraphs.  Thus,
+Interrupting a paragraph with an ordered list introduces an parsing
+ambiguity. What looks like an ordered marker might instead be a
```

Comment:
```suggestion
ambiguity.  What looks like an ordered marker might instead be a
```

Reply:
?

Reply:
ok, I see now that the spec *usually* uses two spaces between sentences. Sure.